lib/Doctrine/ORM/Persisters/Exception/UnrecognizedField.php

```diff
@@ -10,8 +10,8 @@
 
 final class UnrecognizedField extends PersisterException
 {
-    public static function byName(string $field): self
+    public static function byName(string $class, string $field): self
```

Comment:
While I think this BC break is OK, the name no longer fits. Let's create another method called `byEntityAndName`? Or `byFullyQualifiedName`? Or something else? We would remove the old method in 3.0. Also, let's document the first parameter as `class-string` please.

Reply:
done ! Thanks for the feedback :)